internal/graphicsdriver/metal/mtl/mtl_darwin.go

```diff
@@ -338,6 +337,14 @@ const (
 	CommandBufferStatusError       CommandBufferStatus = 5 // Execution of the command buffer was aborted due to an error during execution.
 )
 
+var (
+	metal                         = purego.Dlopen("Metal.framework/Metal", purego.RTLD_GLOBAL)
+	_MTLCreateSystemDefaultDevice = purego.Dlsym(metal, "MTLCreateSystemDefaultDevice")
+
+	_ = purego.Dlopen("/System/Library/Frameworks/CoreGraphics.framework/Versions/Current/CoreGraphics", purego.RTLD_GLOBAL)
```

Comment:
Why do we need `Dlopen` without using the returning value here? If we expect only side effects, what about moving this into `init` functions?